apps/omg_watcher/test/support/exit_processor/test_helper.ex

```diff
@@ -42,8 +42,15 @@ defmodule OMG.Watcher.ExitProcessor.TestHelper do
     eth_height = Keyword.get(opts, :eth_height, 2)
     exit_id = Keyword.get(opts, :exit_id, @exit_id)
     call_data = %{utxo_pos: enc_pos, output_tx: txbytes}
-
-    event = %{owner: owner, eth_height: eth_height, exit_id: exit_id, call_data: call_data}
+    root_chain_txhash = <<0::256>>
```

Comment:
>  test helper updated to include a transaction hash of `<<0::256>>`. Open question whether we'd like a more advanced logic there in this PR.

I think something like this works: https://github.com/omisego/elixir-omg/blob/master/apps/omg_watcher_info/test/support/factories/data_helper.ex#L28-L30. It's in omg_watcher_info namespace but it's straightforward enough to copy over into a `defp`.

Reply:
Thanks @unnawut. Created a `def` in this file that I also re-used for above. c35bec5